homeassistant/components/netatmo/__init__.py

```diff
@@ -66,6 +83,9 @@
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     """Set up Netatmo from a config entry."""
+    if hass.state == CoreState.not_running:
```

Comment:
We should only check for this if `CONF_WEBHOOK_ID not in entry.data`. And let's add a comment.